src/fragmentarium/application/FragmentSearchService.test.ts

```diff
@@ -1,8 +1,12 @@
 import Promise from 'bluebird'
+import { createScript } from 'fragmentarium/infrastructure/FragmentRepository'
 import { testDelegation, TestData } from 'test-support/utils'
 import FragmentSearchService from './FragmentSearchService'
 
-const resultStub = {}
+const resultStub = {
```

Comment:
Adding `script` here because it's expected in post-fetch processing (`createScript` function)